qa/suites/fs/upgrade/nofs/tasks/0-from/reef.yaml

```diff
@@ -1,9 +1,9 @@
 meta:
 - desc: |
-   install ceph/pacific latest
+   install ceph/reef
 tasks:
 - install:
-    branch: pacific
+    branch: reef
```

Comment:
Side effect of this is that `nofs/tasks/1-upgrade.yaml` does

> ceph osd require-osd-release quincy

which would fail since `require-osd-release` can't be set to a lower release value, thereby failing the test with

> 2024-05-27T06:29:01.724 INFO:teuthology.orchestra.run.smithi145.stderr:Error EPERM: require_osd_release cannot be lowered once it has been set

See: /a/vshankar-2024-05-27_05:52:49-fs-wip-vshankar-testing-20240521.063558-debug-testing-default-smithi/7727604

`require-osd-release` was set to quincy since the upgrade started from `pacific` release, but now `nofs` upgrades from reef/squid to main branch (which is still technically `squid` release), so IMO `require-osd-release` can be set to `squid`.

Reply:
oh no, sorry, this is a mistake, let me fix it real quick

Reply:
@vshankar fixed https://github.com/ceph/ceph/compare/1019009193e309b055162e3b0175f8838993ed7d..1cea3ae75cff9c8fabeb8dddbd9763dacaae8008